docs/markdown/Introduction/media.md

```diff
@@ -4,10 +4,94 @@ slug: "media"
 excerpt: "Learn more about Pants and related topics from these talks, posts and podcasts featuring Pants contributors and users."
 hidden: false
 createdAt: "2021-04-18T17:27:18.361Z"
-updatedAt: "2022-04-05T22:02:38.457Z"
+updatedAt: "2022-09-08T22:02:38.457Z"
```

Comment:
I have no idea. If in-repo changes always override web changes regardless of the metadata, fab.

Reply:
Do you need me to remove the change to this line?

Reply:
This is a good practice to do because it means the Readme.com "Updated 5 days ago" info at the bottom of the page will be accurate. It makes our site look more up-to-date